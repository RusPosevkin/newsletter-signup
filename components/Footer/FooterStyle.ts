import styled from 'styled-components';

export const FooterContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  min-height: 50px;
  border-top: 4px dashed rgb(0, 150, 136);
  color: rgb(0, 150, 136);
  font-size: 1.5rem;
  font-weight: bold;
`;
