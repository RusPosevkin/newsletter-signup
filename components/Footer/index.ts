import FooterComponent from "./FooterStructure";

export default FooterComponent;
