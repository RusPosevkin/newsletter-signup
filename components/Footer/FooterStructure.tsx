import React from 'react'
import { FooterContainer } from './FooterStyle';

const FooterStructure = () => (
  <FooterContainer>
    &copy; 2024
  </FooterContainer>
)

export default FooterStructure;
