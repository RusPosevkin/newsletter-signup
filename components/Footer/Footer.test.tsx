import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import Footer from '.'

test('it should match the snapshot and render', async () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
  expect(container).toBeInTheDocument();
});
