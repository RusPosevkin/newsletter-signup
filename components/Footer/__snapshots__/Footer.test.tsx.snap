// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the snapshot and render 1`] = `
<div>
  <div
    class="sc-1d4bfd53-0 UONRY"
  >
    Â© 2024
  </div>
</div>
`;
